packages/rum-core/src/boot/rumPublicApi.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { ONE_SECOND, RelativeTime, getTimeStamp, display, TimeStamp } from '@datadog/browser-core'
+import { ONE_SECOND, RelativeTime, getTimeStamp, display, TimeStamp, Datacenter } from '@datadog/browser-core'
```

Comment:
what about not importing this type?

Reply:
What is  your reasonning behind it? 
I personally like to have a valid configuration on tests.

Reply:
since we don't support Datacenter anymore on RUM, it could be nice to remove any dependency on it in RUM package.

Reply:
yes, done